meta-mender-qemu/conf/machine/vexpress-qemu-flash.conf

```diff
@@ -13,8 +13,8 @@ IMAGE_FSTYPES = "ubifs vexpress-nor"
 MACHINE_ESSENTIAL_EXTRA_RDEPENDS = "kernel-image kernel-devicetree"
 MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += "mtd-utils mtd-utils-ubifs mtd-utils-jffs2"
 
-MKUBIFS_ARGS = "-m 1 -e 262016 -c 1024"
-UBINIZE_ARGS = "-p 256KiB -m 1 -O 64"
+MKUBIFS_ARGS ?= "-m 1 -e 262016 -c 1024"
```

Comment:
`MKUBIFS_ARGS` and `UBINIZE_ARGS` are probably not something you will need/want to change